config/desktop/jammy/appgroups/net-tools-gnome-gui/packages

```diff
@@ -0,0 +1,11 @@
+network-manager-config-connectivity-ubuntu
+network-manager-fortisslvpn-gnome
+network-manager-iodine-gnome
+network-manager-l2tp-gnome
+network-manager-openconnect-gnome
+network-manager-openvpn-gnome
+network-manager-pptp-gnome
+network-manager-ssh-gnome
+network-manager-sstp-gnome
+network-manager-vpnc-gnome
+
```

Comment:
I hope these are going to be useful, and it didn't remove the previous network stack

I dont like gnome, and I dispise the new network manager (mostly because it ignores `wpa_supplicant.conf` which hold a lot of data for my use case)